tutoraspects/templates/openedx-assets/assets/dashboards/Instructor_Dashboard.yaml

```diff
@@ -2,7 +2,7 @@ _file_name: Instructor_Dashboard.yaml
 _roles:
 - {{ SUPERSET_ROLES_MAPPING.instructor }}
 css: ''
-dashboard_title: Instructor Dashboard
+dashboard_title: {{ SUPERSET_DASHBOARDS.instructor_dashboard.title }}
```

Comment:
Same as above with these changes, I think these shouldn't be settings and instead anything relying on them should have its own separate configuration.